bricksrc/equipment.py

```diff
@@ -92,9 +95,11 @@
             },
             "CWS": {
                 OWL.equivalentClass: BRICK["Chilled_Water_System"],
+                "tags": [TAG.CWS],
```

Comment:
Some thing here as with `HVAC`; do we want to have the tags for the expanded name, or the abbreviation?